schemas/comment.py

```
@@ -0,0 +1,37 @@
+    author: UserId
+    content: Content
+    created_at: datetime
+
+
+class CommentUpdateRequest(BaseModel):
+    content: Content | None = None
+
+    @model_validator(mode="after")
+    def at_least_one_field(self):
+        if self.content is None:
+            raise ValueError("수정할 필드가 없습니다.")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)